src/java.base/share/classes/java/nio/BufferCleaner.java

```diff
@@ -212,6 +220,9 @@ public void run() {
                     Cleaner c = (Cleaner) queue.remove();
                     c.clean();
                 } catch (InterruptedException e) {
+                    if (CleanerShutdown.shuttingDown()) {
+                        break;
+                    }
```

Comment:
I think this should go after the comment (not that it makes any functional difference).

Reply:
Actually, I think the comment should just go: We're not ignoring the interrupt, but instead deciding whether we're done.

Please correct the indentation of line 226.